tests/interfaces/default_qubit_2_integration/test_autograd_qnode.py

```diff
@@ -24,31 +24,31 @@
 from pennylane.tape import QuantumScript
 from pennylane.devices.experimental import DefaultQubit2
 
-qubit_device_and_diff_method = [
-    [DefaultQubit2(), "finite-diff", False],
-    [DefaultQubit2(), "parameter-shift", False],
-    [DefaultQubit2(), "backprop", True],
-    [DefaultQubit2(), "adjoint", True],
-    [DefaultQubit2(), "adjoint", False],
-    [DefaultQubit2(), "spsa", False],
-    [DefaultQubit2(), "hadamard", False],
+diff_method_list = [
```

Comment:
I included the device in these lists so that it would be easy to extend these tests to other devices. For example, we could potentially add in lightning or mixed to the integration tests here.